css-two-column-layout/styles.css

```diff
@@ -0,0 +1,72 @@
+body {
+  box-sizing: border-box;
+  margin: 0 auto;
+  font-family: 'Karla', sans-serif, monospace;
+}
+
+.container {
+  width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  padding-bottom: 20px;
+}
+
+.header {
+  width: 85%;
+  text-align: center;
+  padding: 0 20px;
+  margin: 0 auto;
+}
+
+.header h1 {
+  font-size: 3rem;
+}
+
+.header p {
+  color: rgb(90, 90, 90);
+  font-size: 0.9rem;
+  line-height: 1.8rem;
+}
+
+.column {
+  display: flex;
```

Comment:
`column` classes are supposed to have a `width` or `flex-basis` and nothing else. SInce you set `display: flex` here, that is an indicator that you likely need a `row` inside of the `column` somehow. You should also set a proper width on this `column` class and give it a name that matches that width

Reply:
This still needs to be addressed